libs/placeholder.txt

```diff
@@ -0,0 +1 @@
+Put the umple jar in this folder. It should be named umple-latest.jar.
```

Comment:
We could use `git-lfs` if necessary here, but ideally we need to package umple for ivy/maven and ship it that way.